docs/source/ingest/destination_connectors/weaviate.rst

```diff
@@ -0,0 +1,71 @@
+Weaviate
+===========
+
+Batch process all your records using ``unstructured-ingest`` to store structured outputs locally on your filesystem and upload those local files to a Weaviate collection.
+
+First you'll need to install the weaviate dependencies as shown here.
+
+.. code:: shell
+
+  pip install "unstructured[weaviate]"
+
+Run Locally
+-----------
+The upstream connector can be any of the ones supported, but for convenience here, showing a sample command using the
+upstream weaviate connector. This will push elements into a collection schema of your choice into a weaviate instance
+running locally.
+
+.. tabs::
+
+   .. tab:: Shell
+
+      .. code:: shell
+
+        unstructured-ingest \
+            s3 \
```

Comment:
Can this use the local connector as the source to mirror the ingest destination tests